src/main/js/dashboard-frontend/src/components/Arch.tsx

```diff
@@ -72,20 +72,23 @@ class Arch extends Component {
     ];
 
     return (
-      <Container header={<h2>Device Details</h2>}>
-        <ColumnLayout columns={4} variant="text-grid">
-          {items.map((group) => (
-              <SpaceBetween size="xs">
-              {group.map((item) => (
-                <div>
-                  <Box margin={{bottom: "xxxs"}} color="text-label">{item.field}</Box>
-                  <div>{item.value}</div>
-                </div>
+        <>
+          <Header></Header>
```

Comment:
Is this empty header required?

Reply:
Just to add spacing, I couldn't find a better way to add vertical spacing

Reply:
updated